corehq/apps/hqadmin/migrations/0016_hqdeploy_ordering.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-05-10 00:54
+from __future__ import unicode_literals
```

Comment:
nit: autogenerated cruft (I can't wait 'till Django 2!)

Reply:
81f331ee9ab ...just because you care enough about this that you comment on it in every time.